pyproject.toml

```diff
@@ -29,7 +29,7 @@ repository = "https://github.com/jackmpcollins/magentic"
 [tool.poetry.dependencies]
 python = ">=3.10,<4.0"
 filetype = "*"
-litellm = {version = ">=1.0.0", optional = true}
+litellm = {version = ">=1.31.6", optional = true}
```

Comment:
Required for streamed function calling with Claude 3 model, fixed in litellm release https://github.com/BerriAI/litellm/releases/tag/v1.31.6